models/lead/src/index.ts

```diff
@@ -139,7 +139,8 @@ export function createModel (builder: Builder): void {
             componentProps: {
               _class: lead.mixin.Customer,
               icon: lead.icon.Lead,
-              label: lead.string.Customers
+              label: lead.string.Customers,
+              viewOptionClasses: [contact.class.Organization]
```

Comment:
I don't like this. We just need to discuss and allow display or not display all childs props

Reply:
Question here, is a starting point for filters.